packages/app-review/src/definitions.ts

```diff
@@ -1,3 +1,21 @@
 export interface AppReviewPlugin {
-  echo(options: { value: string }): Promise<{ value: string }>;
+  /**
+   * Open the App Store page for the current app and, if possible, open the dialog to leave a review.
+   *
+   * Only available on Android and iOS.
+   *
+   * @param options The options to open the App Store page, only available on iOS.
+   * @since 6.0.0
+   */
+  openAppStore(options?: { appID: string }): Promise<void>;
```

Comment:
```suggestion
  /**
   * Open the App Store page for the current app and, if possible, open the dialog to leave a review.
   *
   * Only available on Android and iOS.
   *
   * @since 6.0.0
   */
  openAppStore(): Promise<void>;
```

You can just retrieve the app ID programmatically. Example: https://github.com/capawesome-team/capacitor-plugins/blob/d4e76b7f64b165b703667bfb0e5e67e2a543f7e8/packages/app-update/ios/Plugin/AppUpdatePlugin.swift#L19

This means we can omit the parameter, which makes it easier to use.